restructure_by_scope.py

```diff
@@ -0,0 +1,56 @@
+
+"""
+Restructures CredData files into scope-based folders (/src/, /test/, etc.)
+"""
+import os
+import shutil
+from pathlib import Path
+
+# From CredSweeper's ml_config.json
+SCOPES = [
+    "/src/",
+    "/test/",
```

Comment:
Exactly pathes should be used from:
https://github.com/Samsung/CredSweeper/blob/ec247774b70d92e893c37b69e7d0b9dd7bcd6cbc/credsweeper/ml_model/ml_config.json#L353-L363
otherwise, path ``.../stresstest/...`` will be detected as ``/test/`` with the script, but not with CredSweeper.